jvb/src/test/kotlin/org/jitsi/videobridge/MediaSourceFactoryTest.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright @ 2021 - present 8x8, Inc.
+ * Copyright @ 2021 - Vowel, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.videobridge
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.IsolationMode
+import io.kotest.core.spec.Spec
+import io.kotest.core.spec.style.ShouldSpec
+import io.kotest.matchers.shouldBe
+import org.jitsi.config.setNewConfig
+import org.jitsi.videobridge.xmpp.MediaSourceFactory
+import org.jitsi.xmpp.extensions.colibri.SourcePacketExtension
+
+fun createSource(ssrc: Long): SourcePacketExtension {
+    val spe = SourcePacketExtension()
+
+    spe.ssrc = ssrc
+
+    return spe
+}
+
+// TODO port MediaSourceFactoryTest to kotlin and unify with this class
```

Comment:
I think this has been done?

Reply:
uhm no, there's still the java version here: https://github.com/jitsi/jitsi-videobridge/blob/9cbc86449e7b0bfe81b23f835fc6eeb8d1db75bc/jvb/src/test/java/org/jitsi/videobridge/xmpp/MediaSourceFactoryTest.java#L28